testworld/park_test.go

```diff
@@ -33,3 +36,42 @@ func TestHost_Happy(t *testing.T) {
 	getDocumentAndCheck(charlie.httpExpect, charlie.id.String(), typeInvoice, params)
 	fmt.Println("Host test success")
 }
+
+func TestHost_RestartWithAccounts(t *testing.T) {
+	t.Parallel()
+	// Name can be randomly generated
+	tempHostName := "Sleepy"
+	bootnode, err := doctorFord.bernard.p2pURL()
+	assert.NoError(t, err)
+	sleepyHost := doctorFord.createTempHost(tempHostName, doctorFord.twConfigName, defaultP2PTimeout, 8088, 38208, true, true, []string{bootnode})
+	doctorFord.addNiceHost(tempHostName, sleepyHost)
```

Comment:
why do you need to add it here?

Reply:
you can just not add it, since its not useful to be managed by doctorFord, 
the test can create, start and then kill it at the end.

Reply:
right now is needed as if there is no nice hosts then `getHostTestSuite` will fail and I do not want to modify more code here.